src/shared/api/MatchMinerAPI.ts

```diff
@@ -0,0 +1,86 @@
+import * as request from 'superagent';
+import { ITrial, ITrialMatch } from "shared/model/MatchMiner.ts";
+import { buildCBioPortalAPIUrl } from "./urls";
+
+/**
+ * Retrieves the trial matches for the query given, if they are in the MatchMiner API.
+ */
+
+const cbioportalUrl = buildCBioPortalAPIUrl('api-legacy/proxy/matchminer/api');
+export async function postMatchMinerTrialMatches(query: object): Promise<Array<ITrialMatch>> {
+    return request.post(cbioportalUrl + '/post_trial_match')
+    .set('Content-Type', 'application/json')
+    .send(query)
+    .then((res) => {
+        const response = JSON.parse(JSON.parse(res.text));
```

Comment:
why do we parse it twice?

Reply:
@zhx828 the fact that we have to parse it all could be some indication of funkiness further up stream.   think superagent should parse it automatically if it detects the right content type.   not super important, but worth checking

Reply:
maybe content type gets screwed upsince we're proxying

Reply:
This is proxy issue and I have fixed it. Now it only need to be parse once.